libs/features/components/elewa-group-image-and-text-banner/src/lib/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'elewa-group-image-and-text-banner',
+  templateUrl: './elewa-group-image-and-text-banner.component.html',
+  styleUrls: ['./elewa-group-image-and-text-banner.component.scss'],
+})
+export class ElewaGroupImageAndTextBannerComponent {
+  @Input() imagePlacement: string
+  @Input() title: string
+  @Input() imageUrl: string
+  @Input() paragraphText: string
```

Comment:
Initialise to empty string/default value i.e.

@Input() paragraphText: string = '';